_posts/2023-06-21-Web-Hosting-Sponsored-by-Emerge.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "CocoaPods Web Hosting now Sponsored by Emerge Tools"
+author: segiddins
+categories: sponsorship cocoapods.org website
+---
+
+I'm very happy to announce that [Emerge Tools](https://www.emergetools.com) is now sponsoring the web hosting costs of CocoaPods!
```

Comment:
nit: maybe not a bad thing, we use "we" below everywhere in the post.

Reply:
I think this is reasonable given that we have this post as a single author 👍🏻